doc/whats_new/v0.21.rst

```diff
@@ -52,6 +52,18 @@ Support for Python 3.4 and below has been officially dropped.
   graph, which would add explicitly zeros on the diagonal even when already
   present. :issue:`12105` by `Tom Dupre la Tour`_.
 
+:mod:`sklearn.neighbors`
+......................
+
+- |Fix| Fixed a bug in :class:`neighbors.NearestNeighbors` calling functions
```

Comment:
It's not a big bug. Let's phrase it as "Methods x x x now raise NotFittedError, rather than AttributeError, when called before fit". Can label it `|API|` too.